inst/rmarkdown/templates/ajs_outbreak/skeleton/skeleton.Rmd

```diff
@@ -78,367 +78,341 @@ for (pkg in required_packages) {
 }
 
 
+
+## Set default options for plots and charts
+
 # set default text size to 16 for plots
 # give classic black/white axes for plots
 ggplot2::theme_set(theme_classic(base_size = 18))
 
-# Set the day that defines the beginning of your epiweek.
-# you can start the week on any day of the week
-# (the ISO standard is to start on Monday) 
-aweek::set_week_start("Monday")
+# sets the theme in ggplot for epicurves
+epicurve_theme <- theme(
+  axis.text.x = element_text(angle = 45, hjust = 1, vjust = 1), 
+  legend.title = element_blank(),
+  panel.grid.major.x = element_line(color = "grey60", linetype = 3),
+  panel.grid.major.y = element_line(color = "grey60", linetype = 3))
+
+# sets the labels in ggplot for the epicurves
+epicurve_labels <- labs(x = "Calendar week", 
+                        y = "Cases (n)", 
+                        title = "Cases by week of onset",
+                        subtitle = sprintf("Source: MSF data from %s", reporting_week)
+                       ) 
+
 ```
 
 
 
+<!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You need to set the week you want to report on. Generally, this is the previous
+week. Put it below.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
 
 ```{r define_current_week}
 
-# In order to automate your report you need define week of interest (reporting_week)
-# This will generally be one week in the past from when you write your report
+# Set the day that defines the beginning of your epiweek.
+# (the standard is to start on Monday)
+aweek::set_week_start("Monday")
 
-# define current week 
+# set current week 
 reporting_week <- "2019-W25"
-
 ```
 
 
 
-
+<!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To play with this template, you can create fake data. Comment out this chunk
```

Comment:
We discussed this during our call and came to the conclusion that it's better to comment out code because it's easier for the user to spot and correct.